src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DoubleInput.cs

```diff
@@ -28,7 +28,7 @@ public void CustomizeView(DoubleInput nodeModel, NodeView nodeView)
 
             tb.DataContext = nodeModel;
 
-            tb.BindToProperty(new Binding("Value")
+            tb.BindToProperty(new Binding("NumericalValue")
             {
                 Mode = BindingMode.TwoWay,
                 Converter = new DoubleInputDisplay(),
```

Comment:
I wasn't aware of the `ValidationRule` class. That sounds like a better solution than `IValueConverter` and will achieve similar gains. Go for it!